tests/llm-as-a-judge/correctness/test_openai_eval.py

```
@@ -0,0 +1,273 @@
+    if MODEL_RUNNER == "ollama":
+        try:
+            res = subprocess.run(
+                ["ollama", "run", OLLAMA_MODEL],
+                input=prompt.encode("utf-8"),
+                stdout=subprocess.PIPE,
+                stderr=subprocess.PIPE,
+                check=True,
+            )
+            return res.stdout.decode("utf-8").strip()
+        except subprocess.CalledProcessError as e:
+            raise RuntimeError(f"Ollama call failed: {e.stderr.decode('utf-8')}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)